src/results/proforma.jl

```diff
@@ -195,20 +259,8 @@ function proforma_results(p::REoptInputs, d::Dict)
 
         annual_income_from_host_series = repeat([-1 * r["annualized_payment_to_third_party"]], years)
 
-        if "Generator" in keys(d) && d["Generator"]["size_kw"] > 0
-            generator_fuel_cost_series = escalate_om(-1 * d["Generator"]["year_one_fuel_cost_before_tax"])
-            if p.s.generator.existing_kw > 0
-                existing_genertor_fuel_cost_series = escalate_om(-1 * d["Generator"]["year_one_fuel_cost_before_tax_bau"])
-            else
-                existing_genertor_fuel_cost_series = zeros(years)
-            end
-        else
-            existing_genertor_fuel_cost_series = zeros(years)
-            generator_fuel_cost_series = zeros(years)
-        end
-        net_energy_costs = -electricity_bill_series_bau - export_credit_series_bau + electricity_bill_series + 
-                           export_credit_series + annual_income_from_host_series - existing_genertor_fuel_cost_series + 
-                           generator_fuel_cost_series
+        net_energy_costs = -electricity_bill_series_bau - export_credit_series_bau - m.fuel_cost_series_bau + electricity_bill_series + 
```

Comment:
It looks like `net_energy_costs` only gets used in `deductable_net_energy_costs`, which does not get used anywhere else. Could we remove this code altogether?

Reply:
Yep.